### ISSUE TITLE
Add the option to print the lesson

### ISSUE THREAD

**Issue by henspace** [henspace/text2lesson#41]
From the LessonPresenter, it would be nice to be able to print a hardcopy of the lesson. This should produce a worksheet that could be used as a printed lesson. The option to print in a large font for accessibility should be provided.
